common-content/modules/ROOT/partials/client-applications.adoc

```diff
@@ -1070,7 +1070,7 @@ include::{python-examples}/test_kerberos_auth_example.py[tags=kerberos-auth]
 
 [NOTE]
 --
-The Kerberos authentication token can only be understood by the server if the server has the link:{kerberos-add-on-base-uri}[Kerberos Add-on] installed.
+The Kerberos authentication token can only be understood by the server if the server has the link:/docs/kerberos-add-on/current/[Kerberos Add-on] installed.
```

Comment:
```suggestion
The Kerberos authentication token can only be understood by the server if the server has the link:{neo4j-docs-base-uri}/kerberos-add-on[Kerberos Add-on] installed.
```

The `neo4j-docs-base-uri` has just been added (this morning!) and will always resolve correctly whether you are building locally with _preview.yml_ or publishing with _publish.yml_.